lib/helpers/sfConnection.js

```diff
@@ -0,0 +1,25 @@
+const jsforce = require('jsforce');
+const common = require('../common.js');
+
+
+exports.createConnection = function createConnection(configuration, emitter) {
```

Comment:
Is it possible to convert it to `async`?

Reply:
All we need is return `connection` object. And we get it without any asynchronous code. So we don't need here `async`